scripts/devops_tasks/verify_autorest.py

```diff
@@ -54,7 +55,31 @@ def find_swagger_folders(directory):
     return ret
 
 
-def check_diff(folder):
+def check_for_open_pr(service_directory):
```

Comment:
You shouldn't need to do this if your branch name is consistent. As the create-pull-request template will not create an additional PR against a given branch if one already exists. That is how we generally handle this case in other places.

Reply:
That is great to know, I can close this PR entirely then